src/include/daos/security.h

```diff
@@ -10,7 +10,16 @@
 #include <stdint.h>
 #include <stddef.h>
 #include <daos_types.h>
+#include <daos_prop.h>
 
+/**
+ * Structure representing a resource's ownership by user and group,
+ * respectively.
+ */
```

Comment:
I think this would probably fit on one line given our 100 character line length.
```suggestion
/** Structure representing a resource's ownership by user and group, respectively. */
```